config/database.php

```diff
@@ -79,7 +79,7 @@
             'password' => env('DB_PASSWORD', 'root'),
             'unix_socket' => env('DB_SOCKET', ''),
             'charset' => 'utf8mb4',
-            'collation' => 'utf8mb4_unicode_ci',
```

Comment:
Just curious: What does this actually do in practice and why null?

Reply:
https://github.com/laravel/laravel/pull/6241